modules/idle-unidling-applications.adoc

```diff
@@ -25,3 +25,8 @@ $ oc scale --replicas=1 dc <dc_name>
 Automatic unidling by a router is currently only supported by the default
 HAProxy router.
 ====
+[NOTE]
+====
+Automatic unidling by a service is not supported when Kuryr-Kubernetes is
```

Comment:
"unidling" is definitely the correct jargon to use here? Would love if it weren't, but if it is, it is what it is. :) @dulek

Reply:
It seems so, judging from the rest of the module I'm modifying. The note I'm adding is analogous to the one above.